app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="card-columns">
+  <% @categories.each do |category| %>
+    <%= link_to category_path(category) do %>
+      <div class="container card-container">
+        <img src="<%= category.icon %>" class="card-image" alt="Category Icon">
+        <div class="card-contents">
+          <h5 class="card-name"><%= category.name %></h5>
+          <p class="card-date"><%= category.created_at.strftime('%d %B, %Y') %></p>
+        </div>
+        <p class="total-cash">$<%= @category_totals[category.id] %>
+        </p>
+      </div>
+    <% end %>
+  <% end %>
+  <div class="container add-category">
+    <%= link_to "Add New Category", new_category_path, class: "btn btn-default" %>
+  </div>
```

Comment:
- [ ] Kindly consider aligning with the design guideline and make the design match as possible by putting the button for adding a category at the bottom of the screen and applying the same for adding a new transaction button

Reply:
> * [ ]  Kindly consider aligning with the design guideline and make the design match as possible by putting the button for adding a category at the bottom of the screen and applying the same for adding a new transaction button

# Greetings sir!, I would like  kindly to tell you that you just reviewed an empy Home(varegory) and Transaction pages where they were only buttons but no category lists and transaction lists which  means that the pages were empty but only buttons , and here there is a screen shoot of my pages when you try to add categories and transaction lists you will automatically find the buttons on bottom of the pages
![transaction](https://github.com/ismailmunyentwari9/transact_track/assets/79504482/52738af8-bcd7-4247-b42c-673e972404ac)
![category](https://github.com/ismailmunyentwari9/transact_track/assets/79504482/2eb7a27b-7d44-4bcc-ba1e-0c7792af88e9)